### PR TITLE
ngctl: Nebula Graph Kubernetes Control

### PR DESCRIPTION
**Pull request by veezhang**
- [x] cmd framework
- [x] version
- [x] use
- [x] list
- [x] info
- [x] console
- [ ] debug
- [ ] doc

### CODE REVIEW

**Comment by veezhang** [comment]
Fix: #32

File: pkg/ngctl/cmd/cmd.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	"github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/version"
+)
+
+const (
+	nkcLongDescription = `
+		"ngctl"(NebulaGraph Kubernetes Control) controls the Nebula cluster manager on Kubernetes.
+`
+)
+
+func NewNkctlCmd(in io.Reader, out, err io.Writer) *cobra.Command {
```
Comment (kqzh) [inline pkg/ngctl/cmd/cmd.go:42]: rename this function like NewNgctlCmd will be better

Comment (veezhang) [reply pkg/ngctl/cmd/cmd.go:42]: Emm, u r right!

File: pkg/ngctl/cmd/version/version.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+		IOStreams: ioStreams,
+	}
+}
+
+// NewCmdVersion returns a cobra command for fetching versions
+func NewCmdVersion(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	o := NewOptions(ioStreams)
+	cmd := &cobra.Command{
+		Use:     "version",
+		Short:   "Print the cli and nebula operator version",
```
Comment (kqzh) [inline pkg/ngctl/cmd/version/version.go:64]: use Long: "Print the cli and nebula operator version", it's better than Long == ""

**Review by kqzh — APPROVED** [review]

**Comment by wey-gu** [comment]
Cheers!!!

**Pull request merged**